lighthouse-ci/src/report/report.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const childProcess = require('child_process');
+const ApiClient = require('../server/api/client.js');
+const {getSavedLHRs} = require('../shared/saved-reports.js');
+
+/**
+ * @param {import('yargs').Argv} yargs
+ */
+function buildCommand(yargs) {
+  return yargs.options({
+    token: {type: 'string', required: true},
+    serverBaseUrl: {
+      description: 'The base URL of the server where results will be saved.',
+      default: 'http://localhost:9001/',
+    },
+  });
+}
+
+/**
+ * @return {string}
+ */
+function getCurrentHash() {
+  const result = childProcess.spawnSync('git', ['rev-parse', 'HEAD'], {encoding: 'utf8'});
+  if (result.status !== 0) throw new Error('Unable to determine current hash');
```

Comment:
Unable to determine current hash with `git rev-parse HEAD`

Reply:
👍